test/core/profile.jl

```diff
@@ -53,25 +53,25 @@ end
 let
     str = string(CUDA.@profile trace=true raw=true @cuda identity(nothing))
 
-    @test occursin("cuLaunchKernel", str)
-    @test occursin("identity()", str)
+    @test_broken occursin("cuLaunchKernel", str)
+    @test_broken occursin("identity()", str)
```

Comment:
Why?

Reply:
I was marking things broken to figure out what just doesn't work with the new PT 3.0. Let me mark this PR draft.